contrib/kind-common.sh

```
@@ -1582,24 +1606,57 @@ EOF
+        fi
+      fi
+    fi
+
     echo "Adding routes for Kubernetes pod networks..."
-    NODES=$(kubectl get nodes -o jsonpath='{.items[*].metadata.name}')
+    NODES=$("${kubectl_cmd[@]}" get nodes -o jsonpath='{.items[*].metadata.name}')
     echo "Found nodes: $NODES"
     for node in $NODES; do
       # Get the addresses
-      node_ips=$(kubectl get node $node -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
+      node_ips=$("${kubectl_cmd[@]}" get node $node -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)